imagersite/photo/models.py

```diff
@@ -0,0 +1,44 @@
+from __future__ import unicode_literals
+from django.db import models
+from django.contrib.auth.models import User
+from django.utils.encoding import python_2_unicode_compatible
+
+
+@python_2_unicode_compatible
+class Photos(models.Model):
+    image = models.ImageField(upload_to='photo_files/%y-%m-%d')
+    user = models.ForeignKey(User, null=False)
+    title = models.CharField(max_length=256)
+    description = models.TextField()
+    published = models.CharField(
+        max_length=256,
+        choices=(('private', 'private'), ('public', 'public')),
+        default='private'
+    )
+    date_uploaded = models.DateField(auto_now_add=True)
+    date_modified = models.DateField(auto_now=True)
+    date_published = models.DateField(auto_now_add=True)
```

Comment:
This would be better reserved as something that gets set when a photo's 'published' field is set to 'public'